rust/hacl-rust-sys/src/bindings/bindings.rs

```diff
@@ -33,6 +33,9 @@ pub const EverCrypt_Error_InvalidKey: u32 = 2;
 pub const EverCrypt_Error_AuthenticationFailure: u32 = 3;
 pub const EverCrypt_Error_InvalidIVLength: u32 = 4;
 pub const EverCrypt_Error_DecodeError: u32 = 5;
+pub type __uint8_t = ::std::os::raw::c_uchar;
+pub type __uint32_t = ::std::os::raw::c_uint;
+pub type __uint64_t = ::std::os::raw::c_ulong;
```

Comment:
This can be problematic? https://doc.rust-lang.org/std/os/raw/type.c_ulong.html

Reply:
> This is problematic?

I'm not sure I see an issue with this?
In any case. If you think we should improve the bindings, we can talk about contributing to bindgen. But I won't start changing any of this by hand unless there are known bugs in bindgen.